plrust/src/plrust_proc.rs

```diff
@@ -105,19 +99,55 @@ pub(crate) fn load(pg_proc_oid: pg_sys::Oid) -> eyre::Result<UserCrate<StateLoad
     Ok(loaded)
 }
 
-/// helper function to build a vec of Spi arguments to be used as the composite primary key
-/// `plrust.plrust_proc` needs to locate a function
+// helper function to build the primary key values used to query `plrust.plrust_proc` via Spi
+#[rustfmt::skip]
 #[inline]
 fn pkey_datums(pg_proc_oid: pg_sys::Oid) -> Vec<(PgOid, Option<pg_sys::Datum>)> {
     vec![
-        (PgBuiltInOids::REGPROCOID.oid(), pg_proc_oid.into_datum()),
-        (
-            PgBuiltInOids::TEXTOID.oid(),
-            get_target_triple().into_datum(),
-        ),
+        get_fn_identity_datum(pg_proc_oid),
+        (PgBuiltInOids::TEXTOID.oid(), get_target_triple().into_datum()),
     ]
 }
 
+// helper function to build the function identity (oid, value) datum
+fn get_fn_identity_datum(pg_proc_oid: pg_sys::Oid) -> (PgOid, Option<pg_sys::Datum>) {
+    let oa = pg_sys::ObjectAddress {
+        classId: pg_sys::ProcedureRelationId,
+        objectId: pg_proc_oid,
+        objectSubId: 0,
+    };
+    let identity_ptr = unsafe {
+        #[cfg(feature = "pg13")]
```

Comment:
I assume this should be <=13 here, not sure if that's possible? Or does PL/Rust only support >= 13

Reply:
Only >=13.